src/components/dialogs/payment-complete.vue

```diff
@@ -131,13 +130,13 @@ export default class PaymentCompleteDialog extends Mixins(
       await this.fetchNrPayment(nameReqId, paymentId)
       const { getNrId, paymentStatus, sbcPaymentStatus } = this
       if (sbcPaymentStatus === SbcPaymentStatus.COMPLETED && paymentStatus === PaymentStatus.CREATED) {
-        await this.completePayment(nameReqId, paymentId, this.sessionPaymentAction)
+        await this.setCompletePayment(nameReqId, paymentId, this.sessionPaymentAction)
       }
     }
   }
 
-  async completePayment (nrId: number, paymentId: number, action: string) {
-    const result: NameRequestPayment = await this.setCompletePayment({ nrId, paymentId, action })
+  async setCompletePayment (nrId: number, paymentId: number, action: string) {
+    const result: NameRequestPayment = await NamexServices.completePayment(nrId, paymentId, action)
```

Comment:
This change needs to be verified (tested).

Reply:
(I've had issues before with passing parameters vs object.)

Reply:
this is part of what was tested extensively before going to prod